src/Microsoft.DotNet.Build.Tasks/Targets/publishtest.targets

```diff
@@ -74,8 +74,15 @@
       <Output TaskParameter="Filtered" ItemName="_TestCopyLocalByFileNameWithoutDuplicates" />
     </RemoveDuplicatesWithLastOneWinsPolicy>
 
-    <!-- NOTE: UseHardLinksIfPossible is hard-coded to true because we copy tons of the 
-         same files for every single test project here. -->
+    <!-- Ideally, we'd have UseHardLinksIfPossible on by default because we copy tons of the
+         same files for every single test project here, but it doesn't currently work x-plat.
+         so we only turn it on by default for windows builds for now. -->
+    <PropertyGroup>
+      <CreateHardLinksForCopyTestToTestDirectoryIfPossible Condition="'$(CreateHardLinksForCopyTestToTestDirectoryIfPossible)'=='' and '$(OS)' == 'Windows_NT'">true</CreateHardLinksForCopyTestToTestDirectoryIfPossible>
+      <CreateHardLinksForCopyTestToTestDirectoryIfPossible Condition="'$(CreateHardLinksForCopyTestToTestDirectoryIfPossible)'==''">$(CreateHardLinksForCopyFilesToOutputDirectoryIfPossible)</CreateHardLinksForCopyTestToTestDirectoryIfPossible>
+   </PropertyGroup>
+    <!-- Ideally, we'd have UseHardLinksIfPossible on by default because we copy tons of the 
+         same files for every single test project here, but it doesn't currently work x-plat. -->
```

Comment:
Probably don't need the comment twice, and once you're editing it, you could capitalize "Windows" on line 79.